chain/store/messages.go

```diff
@@ -106,6 +106,10 @@ type BlockMessages struct {
 func (cs *ChainStore) BlockMsgsForTipset(ctx context.Context, ts *types.TipSet) ([]BlockMessages, error) {
 	// returned BlockMessages match block order in tipset
 
+	if entry, ok := cs.tsMsgCace.Get(ts); ok {
```

Comment:
Typo?

```suggestion
	if entry, ok := cs.tsMsgCache.Get(ts); ok {
```

Reply:
This won't work with a tipset pointer as a key. See the other PR. We'll probably want the TIpsetKey.